Tests/Integration/DeduplicationWithRedisIntegrationTests.cs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tests.Integration
```

Comment:
Would be best to enable nullable context for new code files:

```suggestion
#nullable enable

namespace LoRaWan.Tests.Integration
```